From: Somnath Kotur <somnath.kotur@emulex.com>
Date: Fri, 30 Sep 2011 07:23:35 +0000
Subject: [PATCH] be2net: Change the data type of the 'on die temperature' stat.
References: bnc#722092
Patch-mainline: v3.1-rc7
Git-commit: 9ae081c66eb8ba0fdb38cca180952782a8744da4

This was showing up as junk value on PPC /Big endian machines since
it was marked as a byte.

Signed-off-by: Somnath Kotur <somnath.kotur@emulex.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/benet/be.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/benet/be.h
+++ b/drivers/net/benet/be.h
@@ -246,7 +246,7 @@ struct be_rx_obj {
 };
 
 struct be_drv_stats {
-	u8 be_on_die_temperature;
+	u32 be_on_die_temperature;
 	u32 tx_events;
 	u32 eth_red_drops;
 	u32 rx_drops_no_pbuf;
